src/service.js

```diff
@@ -202,10 +184,10 @@ function createService(apiClient, { fetchUsers }) {
     createExtension,
     updateExtension,
     deleteExtension,
+    downloadBlobAs,
```

Comment:
You don't need to return this method here because it won't be used elsewhere.
(see also comment above about this method).